modules/ROOT/pages/java-version-support.adoc

```diff
@@ -39,11 +39,31 @@ public class DatabaseConnector {
 }
 ----
 
-You can mark your custom connector as compatible with Java 17 only; however, you must ensure that no adoption or backward compatibility issues exist. 
+[[bytecode_level]]
```

Comment:
same as with the other PR. Customers should not meddle with this. Use parent pom versions exclusively